Console/UpgradeHelperCommand.php

```diff
@@ -32,12 +44,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $diff = file($input->getArgument('diff'));
         $result = $this->runner->run($diff);
 
+        $fileDirectoryPath = $this->directoryList->getPath(DirectoryList::ROOT);
+        $fileName = 'UpgradeHelperResult_'.date('m-d-Y').'.csv';
+        $filePath = $fileDirectoryPath . '/' . $fileName;
+        $data['headers'] = ['Patched', 'Customized', 'Reviewed (Yes / No)', 'Notes'];
```

Comment:
Let's make the columns "Type", "Patched" and "Customized". The other two columns shouldn't be in the CSV output as they're unique to our review process, not generally a concern of this module.